frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -768,19 +743,17 @@ const ManagePolicyPage = ({
           </div>
           {showCtaButtons && (
             <div className={`${baseClass} button-wrap`}>
-              {canManageAutomations &&
-                automationsConfig &&
-                hasPoliciesToAutomateOrDelete && (
-                  <div className={`${baseClass}__manage-automations-wrapper`}>
-                    <Dropdown
-                      className={`${baseClass}__manage-automations-dropdown`}
-                      onChange={onSelectAutomationOption}
-                      placeholder="Manage automations"
-                      searchable={false}
-                      options={getAutomationsDropdownOptions()}
-                    />
-                  </div>
-                )}
+              {showAutomationsDropdown && (
```

Comment:
Updated to follow existing patterns of multiple requirements (`canManageAutomations && automationsConfig && hasPoliciesToAutomateOrDelete`) to be defined as a separate constant (`showAutomationsDropdown`).